src/main/java/org/sqlite/SQLiteConfig.java

```diff
@@ -426,6 +426,7 @@ public enum Pragma {
                 "Limit the size of rollback-journal and WAL files left in the file-system after transactions or checkpoints",
                 null),
         LEGACY_FILE_FORMAT("legacy_file_format", "No-op", OnOff),
+        LEGACY_ALTER_TABLE("legacy_alter_table", "Use legacy alter table behavior", OnOff),
```

Comment:
should be placed before `LEGACY_FILE_FORMAT` as to respect alphabetical ordering